### PR TITLE
WD-16705 Set up base template and flask app

### PR DESCRIPTION
**Pull request by immortalcodes**
## Description

- Initiated flask base app
- Created base template file and fixed index file
- Set up Vanilla styling
- Converted hack.css to styles.scss
- Build script for css
- cleaned up the file structure
- Transfered the old pages to `old-templates` folder

## QA
- Follow the steps in the readme to set up env and run the site.
- Go through the files to see all is good and correctly initiated.

## Fixes

- [WD-16705](https://warthogs.atlassian.net/browse/WD-16706)
- [WD- 16706](https://warthogs.atlassian.net/browse/WD-16706)

[WD-16705]: https://warthogs.atlassian.net/browse/WD-16705?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,32 @@
+bleach==6.2.0
```
[the remark this answers is not in the document]
Comment (immortalcodes) [reply requirements.txt:1]: I think these are in-direct dependencies for canonical libraries, I just did a pip freeze in my venv

File: README.md [hunk 2]
```diff
@@ -4,15 +4,15 @@ This is the codebase for the canonical.design static website
...
 ## Development
 
-The simplest way to run the site is with [docker](https://docs.docker.com/engine/install/ubuntu/):
+The simplest way to run the site is:
 
-```bash
-docker build --tag canonical.design .
-docker run -p 8044:80 canonical.design
-```
+- Create a python virtual environment
```
[the remark this answers is not in the document]
Comment (immortalcodes) [reply README.md:12]: It was out of the scope for now, using dotrun and setting up other tools is part of other tasks.

File: README.md [hunk 3]
```diff
@@ -4,15 +4,15 @@ This is the codebase for the canonical.design static website
...
+- install requirements
+- install node modules
+- run `yarn build`
+- run `flask run` in webapp folder
 
-Afterwards the website will be available at <http://localhost:8044>.
+Afterwards the website will be available at <http://localhost:5000>.
```
[the remark this answers is not in the document]
Comment (immortalcodes) [reply README.md:18]: Yes, this is what flask chooses for itself, once we have dotrun. Way of things will be different

File: .gitignore [hunk 4]
```diff
@@ -0,0 +1,68 @@
+# [generated] Bracketed sections updated by Yeoman generator
+# generator-canonical-webteam@3.4.1
+
+# [os] OS & editor files
```
[the remark this answers is not in the document]
Comment (immortalcodes) [reply .gitignore:4]: I agree but we are yet to add all functionalities that might need their share of .gitignore
I can trim it down for now if that is seems better.

File: requirements.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (akbarkz) [reply requirements.txt:1]: agreed, only leave those that are needed now, the rest can be added later

**Review by akbarkz — APPROVED** [review]

**Pull request merged**